addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -841,6 +841,7 @@
                                     <field name="amount_tax_company" optional="hide" readonly="1"/>
                                     <field name="total_amount" options="{'currency_field': 'currency_id'}" string="Subtotal In Currency " optional="show" readonly="product_has_cost" groups="base.group_multi_currency"/>
                                     <field name="total_amount_company" string="Subtotal"/>
+                                    <field name="sheet_id" column_invisible="True"/>
```

Comment:
This is not necessary
This is a form view of a sheet that already has an id
in js you can get it with `this.env.model.root.resId`